webpack/assets/javascripts/react_app/components/HostDetails/index.js

```diff
@@ -93,7 +93,7 @@ const HostDetails = ({
               <Badge key={21}>{response.architecture_name}</Badge>
             </GridItem>
             <GridItem span={2}>
-              <ActionsBar hostName={response.name} />
+              <ActionsBar hostName={id} permissions={response.permissions} />
```

Comment:
Should we rename `hostName` to `hostId`?
IMHO it should be done, as then it might get quite confusing, I like when names are documentation as well and this would quite break my reading of `ActionsBar` component

Reply:
@ezr-ondrej  agree, done :)